### PR TITLE
Added const ref params, remove std::move() with const ref, replace push_back on emplace_back

### PR DESCRIPTION
**Pull request by GermanAizek**

### CODE REVIEW

File: src/logdata/src/logdata.cpp [hunk 1]
```diff
@@ -419,8 +419,8 @@ std::vector<QString> LogData::getLinesFromFile( LineNumber firstLine, LinesCount
 
         processedLines.reserve( decodedLines.size() );
 
-        for ( auto&& line : decodedLines ) {
-            processedLines.push_back( processLine( std::move( line ) ) );
+        for ( const auto& line : decodedLines ) {
+            processedLines.emplace_back( line );
```
Comment (variar) [inline src/logdata/src/logdata.cpp:423]: Before pushing result back the line needs to be passed to `processLine` callable argument

**Review by variar — CHANGES_REQUESTED** [review]
Great, thank you! This reduces the mess accumulated over the years. The build fails in CI because the call to processLine seems to be lost. See my comment.

File: src/logdata/src/logdata.cpp (hunk 1 above)
Comment (GermanAizek) [reply src/logdata/src/logdata.cpp]: @variar, fix it

Comment (variar) [reply src/logdata/src/logdata.cpp]: @GermanAizek I don't see any new commits in PR. Did you push the fix?

Comment (GermanAizek) [reply src/logdata/src/logdata.cpp]: @variar, I made a force push of the last commit by overwriting it

**Pull request merged**